yarn.lock

```diff
@@ -6771,13 +6771,6 @@ ember-concurrency-decorators@^2.0.0:
     ember-cli-typescript "^3.1.3"
     ember-inflector "^3.0.1"
 
-ember-diff-attrs@^0.2.2:
```

Comment:
Was this also removed from package.json? If not, we should remove it.

Reply:
This was also removed from the package.json.